_pytest/runner.py

```diff
@@ -134,7 +135,11 @@ def _update_current_test_var(item, when):
     """
     var_name = 'PYTEST_CURRENT_TEST'
     if when:
-        os.environ[var_name] = '{0} ({1})'.format(item.nodeid, when)
+        value = _ascii_escaped('{0} ({1})'.format(item.nodeid, when))
```

Comment:
Sorry, I didn't push the changes yet (I wrote my comment meaning to push right away, but got dragged by other things).